students/hyeonjin/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+# Create your models here.
+
+
+class User(models.Model):
```

Comment:
```suggestion
from django.db import models

class User(models.Model):
```

불필요한 공백 및 주석은 지워주세요~!